python/tests/test_df.py

```diff
@@ -81,7 +81,7 @@ def test_limit(df):
 
 def test_udf(df):
     # is_null is a pa function over arrays
-    udf = f.udf(lambda x: x.is_null(), [pa.int64()], pa.bool_())
+    udf = f.udf(lambda x: x.is_null(), [pa.int64()], pa.bool_(), f.Volatility.immutable())
```

Comment:
I believe this line is causing the python code formatter to fail 

You can fix it by running a command such as the following

```
cd datafusion
black --line-length 79 python
```